util/generate_all

```diff
@@ -6,6 +6,28 @@ if [ ! -f util/.serverpod_util_root ]; then
     exit 1
 fi
 
+generateServerUpgradeFiles() {
```

Comment:
Question: What does this do and why is it required?

Reply:
We need to generate a different `serverpod_test_tools.dart` file for the upgraded project. But to run the generate command all project files needs to be present. So I have to merge over some project files temporarily to be able to generate, and then delete everything but the generated `serverpod_test_tools.dart`. I just went for a naive solution, but maybe there is a more elegant solution?
 
 By your question I take it I should add a documenting comment on this one.

Reply:
46f5dff

Reply:
Also per offline discussion, added a test for `dart test` after upgrade: 9726780